exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = true; // finish this statement
```

Comment:
I think the idea here was to use the comparison operators (like >, <, >=, <=, ===) to complete the statement. You could reference the variables that are already being initialised here (studentCount and mentorCount) - so even if the values of those variables change, moreStudentsThanMentors will still have the correct value.